weave/trace_server/remote_http_trace_server.py

```diff
@@ -54,13 +59,34 @@ def from_env(cls, should_batch: bool = False) -> "RemoteHTTPTraceServer":
     def set_auth(self, auth: t.Tuple[str, str]) -> None:
         self._auth = auth
 
-    def _flush_calls(self, batch: t.List) -> None:
+    def _flush_calls(
+        self, batch: t.List, _should_update_batch_size: bool = False
```

Comment:
Do you have a convention around the leading underscore?

Reply:
I use "_" to essentially indicate private / protected:
* Any top-level symbol in a module that is not intended to be consumed externally from that module. Alternative to this is using the `__all__` list of symbol names
* Any class method or property that is not intended to be accessed externally

Then from a parameter perspective, I use it to indicate params that are not intended to be part of a "good" API. Either it is used internally, or (in this case) it is leaking some internal implementation details into the api space. In this case, i could have alternatively dynamically defined a new internal recursive method that effectively hides this.

Reply:
Thanks, just seemed kind of redundant since this is already a private method.